### ISSUE TITLE
tweak resource ids for :outcome-panel/contact-nurse-triage and :outcome-panel/for-further-evaluation

### ISSUE THREAD

**Issue by oakmac** [LuminareMed/quickscreen-translations#11]
Follow up from PR #9 

> Right now, :outcome-panel/contact-nurse-triage and :outcome-panel/for-further-evaluation make up one sentence but are divided (I'm assuming because of the phone number in between the two). In Korean the order needs to be reversed. But in this case it will still be comprehensible as long as the two phrases stay in one line/sentence and the phone number stays where it is.

How can we re-write the text here in order to handle this situation? [tempura](https://github.com/ptaoussanis/tempura) supports some light variable interpolation. Maybe we can do something like:

```edn
:outcome-panel/contact-call-center "Contact the COVID-19 Nurse Triage Call Center at % for further evaluation."
```

Would that work @semper21 ?

**Comment by semper21** [comment]
Yes it would. I can make the changes, but the keys have to be changed for all languages, right?

**Comment by oakmac** [comment]
@semper21 updated the dictionaries to use positional arguments, see https://github.com/LuminareMed/quickscreen-translations/commit/c4b6a041f009cd339394db8605baf8b2c9bc0724

for this issue, see https://github.com/LuminareMed/quickscreen-translations/blob/master/ko.edn#L127-L129

**Comment by oakmac** [comment]
Fixed with PR #13
